app/gather/assets/package.json

```diff
@@ -20,6 +20,7 @@
     "start": "node ./conf/webpack.server.js"
   },
   "dependencies": {
+    "@formatjs/intl-relativetimeformat": "^4.2.0",
```

Comment:
Note: we use the `~` modifier because we faced in the past breaking changes within minor releases, from `1.2.0` to  `1.3.0`. Those kind of errors are not caught in local developments were you barely update your instance but in Travis or in the deployment steps where are executed could lead to unexpected problems.

This decision forces us to check updates more regularly than usual but on the other hand you can expect the same behavior locally and in the server.